staging/cse/windows/provisioningscripts/loggenerator.ps1

```diff
@@ -133,8 +133,32 @@ Collect-OldLogFiles -Folder "C:\k\" -LogFilePattern azure-vnet-ipam.log.*
 
 # Collect running containers
 if ($global:ContainerRuntime -eq "containerd") {
+    Write-Host "Generating containerd-info.txt"
+    containerd.exe --v > (Join-Path $aksLogFolder "containerd-info.txt")

```

Comment:
why could we not check whether the binary exists or not?

Reply:
I think the judge of ContainerRuntime takes the function of checking whether the binary exists or not ?

Reply:
These commands are different from those in collect-windows-logs.ps1. Since this is a regular scheduled task, we should be more careful since it will always fail every N minutes if the binary does not exist.